content/docs/command-reference/pull.md

```diff
@@ -18,40 +18,42 @@ positional arguments:
 
 ## Description
 
-The `dvc pull` and `dvc push` commands are the means for uploading and
-downloading data to and from remote storage. These commands are analogous to
-`git pull` and `git push`, respectively.
+The `dvc push` and `dvc pull` commands are the means for uploading and
+downloading data to and from remote storage (S3, SSH, GCS, etc.). These commands
+are similar to `git push` and `git pull`, respectively.
 [Data sharing](/doc/use-cases/sharing-data-and-model-files) across environments
 and preserving data versions (input datasets, intermediate results, models,
-[metrics](/doc/command-reference/metrics), etc) remotely (S3, SSH, GCS, etc.)
-are the most common use cases for these commands.
+[metrics](/doc/command-reference/metrics), etc.) remotely are the most common
+use cases for these commands.
 
-The `dvc pull` command allows one to retrieve data from remote storage.
-`dvc pull` has the same effect as running `dvc fetch` and `dvc checkout`
-immediately after.
+`dvc pull` downloads data from [remote storage](/doc/command-reference/remote)
+and places it in the <abbr>workspace</abbr>. It has the same effect as running
```

Comment:
`places into workspace` - not a good term, sounds like it literally downloads them into workspace

Reply:
True. Updated this language to be more precise in both refs.